front/components/assistant/AssistantDetails.tsx

```diff
@@ -287,23 +289,52 @@ export function AssistantDetails({
           {otherActions.map((action, index) =>
             isDustAppRunConfiguration(action) ? (
               <div className="flex flex-col gap-2" key={`action-${index}`}>
```

Comment:
You could refactor using the Page component to use SectionHeader for title and so on...

Reply:
Let's finish vault first :)